CareKit/Connect/OCKContact.m

```diff
@@ -134,4 +133,37 @@ - (instancetype)copyWithZone:(NSZone *)zone {
     return contact;
 }
 
+#pragma mark - Monogram
+- (void)setMonogram:(NSString *)monogram {
+    if (!monogram) {
+        monogram = [self generateMonogram:_name];
```

Comment:
Could this be implemented on the getter, not the setter? Then the `self.monogram` "hack" in the initializer is not needed.